docs/HOWTO/90_Misc/Unpack_dvd.rst

```diff
@@ -1,28 +1,43 @@
 .. _howto-unpack-dvd:
 
-HOWTO Extract the Full OS RPM Package Set
-=========================================
+HOWTO Use unpack_dvd to Extract the Full OS RPM Package Set
+===========================================================
 
 The SIMP ISO provides a minimal set of packages.
 
 If you require additional OS packages, you can extract them from vendor ISOs using
-:program:`unpack_dvd`.
+:program:`unpack_dvd`.  Additionally you can extract the extract the PXE boot
+TFTP boot files from the ISO to the rsync directories.
```

Comment:
the TFTP boot files extracted are the kernel and root filesystem files.  They are distinct from the boot files (menu.c32, pxelinux.0, grub.efi, shim.efi etc)